project_4/README.md

```diff
@@ -38,8 +38,8 @@ dmesg //displays the kernel ring buffer
 -Again, use dmesg to check that the list has been removed once the kernel module has been unloaded.
 
 ```
-sudo insmod collatz.ko //loads the "color.ko" kernel module into the Linux kernel , default p=100
-sudo insmod collatz.ko p=11 //loads the "color.ko" kernel module into the Linux kernel , set p=11
+sudo insmod collatz.ko //loads the "color.ko" kernel module into the Linux kernel , default start=100
```

Comment:
type color.ko it is collatz.ko

Reply:
done, sorry i  did not notice this before..